lib/chef/resource/timezone.rb

```diff
@@ -95,12 +95,18 @@ class Timezone < Chef::Resource
                 shell_out!("sudo systemsetup -settimezone #{new_resource.timezone}")
               end
             end
+          when "windows"
+            unless current_windows_tz.casecmp?(new_resource.timezone)
+              converge_by("set timezone to \"#{new_resource.timezone}\"") do
+                shell_out!("tzutil /s \"#{new_resource.timezone}\"")
+              end
+            end
           end
         end
       end
 
       action_class do
-        # detect the current TZ on darwin hosts
+        # detect the current TZ on darwin hosts & windows hosts
```

Comment:
This comment was actually for the macos method. Can you add a comment and YARD tag to the windows method below as well

Reply:
Hi there I've put your tag back the way it was and added one to mine, is that OK?